spec/features/admin_order_view_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'users past order views', type: :feature do
+  let(:admin) { User.create(username: 'admin', password: 'password',
+    password_confirmation: 'password', first_name: 'Justin',
+    last_name: 'Bieber', email: 'asdf@gmail.com',
+    role: 1) }
+  let(:user) { Fabricate(:user) }
+  let(:category) { Fabricate(:category) }
+  let(:item) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.